### PR TITLE
Refactor/branches js

### PR DESCRIPTION
**Pull request by pate**

### CODE REVIEW

**Comment by pate** [comment]
for review. Not precious about this style. Vue.js / React / Mithril. They all do the same thing. Vue.js templating slightly easier here for designers.

File: templates/page.branches.liquid [hunk 1]
```diff
@@ -9,26 +9,27 @@
...
-          $template.find('.branch-phone').text(branch.phone);
-          $template.find('.branch-email').text(branch.email);
-          $('#contacts').append($template)
+    <script type="text/javascript" src="{{ 'branches.js' | asset_url }}"></script>
+    <script src="https://unpkg.com/vue/dist/vue.js"></script>
+
+    <script type="text/javascript">
+      var branches = window.branches;
+      var app4 = new Vue({
+        delimiters: ['${', '}'],
```
Comment (pate) [inline templates/page.branches.liquid]: because {{}} conflict with Liquid

File: templates/page.branches.liquid [hunk 2]
```diff
@@ -9,26 +9,27 @@
       Regular page content goes here.
     {% endcomment %}
 
+    <div id="branches-app">
+        <div v-for="branch in branches">
+          <h2>${ branch.name }</h2>
+          <span>Email: <a href="mailto:${ branch.email }">${ branch.email }</a></span>
```
Comment (pate) [inline templates/page.branches.liquid]: Vue2 is moaning about interpolation. Need to do some bind thing - gross. Mithril might be lighter here. Vanilla React (React.createClass...) can work too.

**Comment by Miguel-P** [comment]
The reason that the CircleCI build is failing is because the python script was altered to run locally. Needs to be changed back.

**Pull request merged**